wdl-lint/src/rules/blank_lines_between_elements.rs

```diff
@@ -44,15 +44,15 @@ fn excess_blank_line(span: Span) -> Diagnostic {
     Diagnostic::note("extra blank line(s) found")
         .with_rule(ID)
         .with_highlight(span)
-        .with_fix("remove the blank line(s)")
+        .with_fix("remove extra blank line(s)")
 }
 
 /// Creates a missing blank line diagnostic.
 fn missing_blank_line(span: Span) -> Diagnostic {
     Diagnostic::note("missing blank line")
         .with_rule(ID)
         .with_highlight(span)
-        .with_fix("add a blank line before this element")
+        .with_fix("add blank line")
```

Comment:
While I think it's okay for the notes to be a bit more terse, I'd encourage keeping a bit more sentence-like structure here. A good litmus test is "would the fix message stand alone as a sentence well"? In the case of `add blank line`, for example, I think it would not (and indeed, I prefer the original fix much more.